src/XrdPfc/XrdPfcIOFile.cc

```diff
@@ -153,11 +153,30 @@ void IOFile::DetachFinalize()
 {
    // Effectively a destructor.
 
-   TRACE(Info, "DetachFinalize() " << this);
+   TRACE(Debug, "DetachFinalize() " << this);
 
    m_file->RequestSyncOfDetachStats();
    Cache::GetInstance().ReleaseFile(m_file, this);
 
+#ifndef NODEBUG
```

Comment:
Please see https://github.com/xrootd/xrootd/issues/794. We are thinking about removing the `NODEBUG` macro altogether for XRootD 6.0, so we should probably not introduce new uses of it. I suggest using the more standard `NDEBUG` macro which is often used to disable debug-only functionality and extra checks for relase builds. In the process of removing `NODEBUG`, I will probably select some things to keep (gated by `#ifndef NDEBUG`) instead of removing everything.

Reply:
Took out the ifdef, this is an important part of the error stream, reporting summary of encountered errors for a whole IO session.